app/views/ips/_table.html.erb

```diff
@@ -1,6 +1,16 @@
 <table class="govuk-table">
   <caption class="govuk-table__caption">
     <%= location.full_address %>
+  <details class="govuk-details govuk-!-margin-bottom-0 govuk-!-margin-top-2">
```

Comment:
Feels like we're using these `!` classes a lot, worry it's going to make things more complicated.

Not sure if there's a way to avoid it, just wanted to share the worry out loud.

Reply:
When messing around with padding and margins its hard not to use them to make the UI look more _professional_

Reply:
Yeah, it's no criticism of how this has ended up looking - think it does good work here - just want to make sure we're not face-kicking future-us.